File: modul3/Python/PRAK305_2310817210017_MUHAMMADNURHUDABAIHAQI.py
jumlah_detik = int(input(""))

hari = jumlah_detik // 86400
jumlah_detik %= 86400
jam = jumlah_detik // 3600
jumlah_detik %= 3600
menit = jumlah_detik // 60
jumlah_detik %= 60
detik = jumlah_detik

if jam >= 24:
    hari = jam // 24
    jam %= 24

if hari > 0:
    print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")

File: modul3/C/PRAK305_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>

int main() {
    int jumlahdetik;

    printf("");
    scanf("%d", &jumlahdetik);

    int hari, jam, menit, detik;
    hari = jumlahdetik / 86400;
    jumlahdetik %= 86400;
    jam = jumlahdetik / 3600;  
    jumlahdetik %= 3600;       
    menit = jumlahdetik / 60;
    jumlahdetik %= 60;
    detik = jumlahdetik % 60;

     if (jam > 24) {
            hari = jam / 24;
            jam %= 24;
        }
        if (hari>0){
            printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, detik);
        } else {
            printf("%02d:%02d:%02d\n", jam, menit, detik);
        }

    return 0;
}
